src/explore-education-statistics-admin/src/pages/publication/PublicationPageContainer.tsx

```diff
@@ -0,0 +1,92 @@
+import Link from '@admin/components/Link';
+import NavBar from '@admin/components/NavBar';
+import Page from '@admin/components/Page';
+import PageTitle from '@admin/components/PageTitle';
+import { PublicationContextProvider } from '@admin/pages/publication/contexts/PublicationContext';
+import {
+  publicationReleasesRoute,
+  PublicationRouteParams,
+} from '@admin/routes/publicationRoutes';
+import publicationService from '@admin/services/publicationService';
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import RelatedInformation from '@common/components/RelatedInformation';
+import useAsyncHandledRetry from '@common/hooks/useAsyncHandledRetry';
+import React from 'react';
+import { generatePath, Route, RouteComponentProps, Switch } from 'react-router';
+
+const navRoutes = [publicationReleasesRoute];
+
+const routes = [...navRoutes];
+
+interface MatchProps {
+  publicationId: string;
+}
```

Comment:
Don't need this as we can use `PublicationRouteParams` instead i.e. `RouteComponentProps<PublicationRouteParams>`